src/pages/HomePage.jsx

```diff
@@ -7,11 +7,12 @@ import TopicBar from "../components/TopicBar";
 import styles from "/Users/alhunt/northcoders/frontend/nc-news/src/styles/ArticleContainer.module.css";
```

Comment:
Can you fix this import to avoid importing from local files. New PR after please!